Makefile

```diff
@@ -12,6 +12,7 @@ LOGLEVEL ?= silent
 OSTYPE := $(shell uname -s | tr '[A-Z]' '[a-z]')
 COVTESTS ?= test
 GTEST_FILTER ?= "*"
+export PYTHON
```

Comment:
Is it necessary to export it? Modifying environment variables as a side effect of `make` may be an unexpected behavior. I'd rather require users to either export it explicitly or prepend each command with `PYTHON=...`.